admin/services/data-access/headteacher-declaration.data.service.js

```diff
@@ -49,4 +49,98 @@ headteacherDeclarationDataService.sqlFindHdfForCheck = async (dfeNumber, checkWi
   return R.head(result)
 }
 
+/**
+ * Find count of pupils blocking hdf submission before check end date
+ * @param dfeNumber
+ * @return {Number}
+ */
+headteacherDeclarationDataService.sqlFindPupilsBlockingHdfBeforeCheckEndDate = async (dfeNumber) => {
+  const sql = `
+    SELECT COUNT(p.id) as pupilsCount
+    FROM [mtc_admin].[pupil] p
+    JOIN [mtc_admin].[school] s ON p.school_id = s.id
+    LEFT JOIN [mtc_admin].[pupilStatus] ps ON (p.pupilStatus_id = ps.id)
```

Comment:
Why would we need to LEFT JOIN to pupilStatus?  Surely it is a normal JOIN as there will always be a pupil status.

Reply:
@jon-shipley copy paste mistake, thanks!